extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -237,7 +237,8 @@ export class AmpA4A extends AMP.BaseElement {
    * @override
    */
   preconnectCallback(unusedOnLayout) {
-    const preconnect = adPreconnect[this.element.getAttribute('type')];
+    const config = adConfig[this.element.getAttribute('type')];
+    const preconnect = config ? config.preconnect : null;
```

Comment:
with `config = null`, do we still provide support for the ad type? what are the cases that `config=null` here?